distributed/benchmarks/client.py

```diff
@@ -78,5 +78,6 @@ def time_trivial_tasks(self,resource,steal_interval):
             assert new_worker.available_resources == rdict
         
         steal_plug = stealing.WorkStealing(client.cluster.scheduler)
-        assert steal_plug._pc.callback_time == steal_interval
+        pc = client.cluster.scheduler.periodic_callbacks["stealing"]
+        assert pc.callback_time == steal_interval
```

Comment:
This code may very well need more changes or other changes outside of these lines may be needed as CI was showing an issue ( https://github.com/dask/dask-benchmarks/issues/48 ).